docs/jsonapi-spec.textile

```diff
@@ -612,7 +673,36 @@ Fields included in the filter clause in addition to the @_id@ are ignored.
 
 h3(#considerationsMultiDocumentFailure). Multi Document Failure Considerations
 
-*TODO* multi document failure considerations 
+Commands that modify multiple documents in a single operation, such as @updateMany@ or @insertMany@, do not perform all of the modification in a single transaction. Instead each document is updated individually and concurrently with other modifications and reads. As a consequence there are two failure modes for multi document updates, and each command specifies if it supports one, or both. 
+
+The two failure modes are "Fail Fast" and "Fail Silently". Where the main different is Fail Fast commands normally return results about individual documents, and Fail Silent commands normally return a summary or count of affected document.
+
+There are two category of storage failures under consideration: 
+
+# *Availability Faults*: These occur when the underlying storage platform is unable to successfully complete an operation due node failures. These errors may be transient and the JSON API may retry operations, if operations still fail after retrying the operation will be considered to have failed. 
+# *Concurrency Faults*: These occur when the a document is under very high concurrenct modification which make it impossible for the JSON API to complete an optimistic "compare-and-set" update. Modification operations read documents from the storage platform and update or delete it only if it has not changed since being read. If the document has changed, the filter that seleted the document is tested again on that document, and the modification is re-tried. Under high contention it is possible for one request to "starve" and be unable to complete.     
+
+In the context of Multi Document Failures we consider a failure (after a configured number of retries) from either of these categories as a failure to complete a command.  
```

Comment:
can we get concrete values for retries for both categories? Maybe we can start with `1` and adapt later..